Jenkinsfile_gc

```diff
@@ -2,6 +2,9 @@ pipeline {
     agent {
         label "linux-arm64-docker"
     }
+    triggers {
+        cron('H * * * *') // Runs the pipeline once every hour.
```

Comment:
```suggestion
        cron('@daily') // Runs the pipeline once every hour.
```

As per my last comment

Reply:
My bad, I missed it. Lets go with the proposed cron trigger expression.